### PR TITLE
Added .gitignore

### PR DESCRIPTION
**Pull request by gilzhaiek**
Added vcs.xml for pycharm
Added TODOs in helloworld.py

### CODE REVIEW

**Comment by gilzhaiek** [comment]
@jonathanmak96 can you comment on my pull request - on the actual source file?

**Review by jonathanksmak — COMMENTED** [review]
Which part should I comment on for the .gitignore?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+# dotenv
+.env
+
+# virtualenv
+.venv
+venv/
+ENV/
+
+# Spyder project settings
+.spyderproject
```
Comment (gilzhaiek) [inline .gitignore:137]: @jonathanmak96 
Here is how I comment on the .gitignore file for example.
You can also add a comment or reply to this one

Comment (jonathanksmak) [reply .gitignore:137]: Okay, I get it now. Thanks!

**Comment by gilzhaiek** [comment]
Good, the last thing is that you need to merge this PR

**Pull request merged**